### PR TITLE
fix(curriculum): replace p tag with h3 tag

### LINKED ISSUE #55736: Web Design, Learn Accessibility by Building a Quiz - Step 34 - no p element
### Describe the Issue

Step 34 is asking to give content ("Question #") to the before pseudo-element of the p element (p::before).  However, there is no p element anywhere in the html.  There was no visible change in the preview after completing the step. Instead of the p element, should it be the h3 element (which is wrapping the question number)?  or was the content "Question #" suppose to have a "sr-only" class, so it also be hidden for accessibility? This also effect all the following step afterwards.

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-accessibility-by-building-a-quiz/step-34

### Your code

```
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <meta name="description" content="freeCodeCamp Accessibility Quiz practice project" />
    <title>Accessibility Quiz</title>
    <link rel="stylesheet" href="styles.css" />
  </head>
  <body>
    <header>
      <img id="logo" alt="freeCodeCamp" src="https://cdn.freecodecamp.org/platform/universal/fcc_primary.svg">
      <h1>HTML/CSS Quiz</h1>
      <nav>
        <ul>
          <li><a href="#student-info">INFO</a></li>
          <li><a href="#html-questions">HTML</a></li>
          <li><a href="#css-questions">CSS</a></li>
        </ul>
      </nav>
    </header>
    <main>
      <form method="post" action="https://freecodecamp.org/practice-project/accessibility-quiz">
        <section role="region" aria-labelledby="student-info">
          <h2 id="student-info">Student Info</h2>
          <div class="info">
            <label for="student-name">Name:</label>
            <input type="text" name="student-name" id="student-name" />
          </div>
          <div class="info">
            <label for="student-email">Email:</label>
            <input type="email" name="student-email" id="student-email" />
          </div>
          <div class="info">
            <label for="birth-date">D.O.B.<span class="sr-only">(Date of Birth)</span></label>
            <input type="date" name="birth-date" id="birth-date" />
          </div>
        </section>
        <section role="region" aria-labelledby="html-questions">
          <h2 id="html-questions">HTML</h2>
          <div class="question-block">
            <h3>1</h3>
            <fieldset class="question" name="html-question-one">
              <legend>
                The legend element represents a caption for the content of its
                parent fieldset element
              </legend>
              <ul class="answers-list">
                <li>
                  <label for="q1-a1">
                    <input type="radio" id="q1-a1" name="q1" value="true" />
                    True
                  </label>
                </li>
                <li>
                  <label for="q1-a2">
                    <input type="radio" id="q1-a2" name="q1" value="false" />
                    False
                  </label>
                </li>
              </ul>
            </fieldset>
          </div>
          <div class="question-block">
            <h3>2</h3>
            <fieldset class="question" name="html-question-two">
              <legend>
                A label element nesting an input element is required to have a
                for attribute with the same value as the input's id
              </legend>
              <ul class="answers-list">
                <li>
                  <label for="q2-a1">
                    <input type="radio" id="q2-a1" name="q2" value="true" />
                    True
                  </label>
                </li>
                <li>
                  <label for="q2-a2">
                    <input type="radio" id="q2-a2" name="q2" value="false" />
                    False
                  </label>
                </li>
              </ul>
            </fieldset>
          </div>
        </section>
        <section role="region" aria-labelledby="css-questions">
          <h2 id="css-questions">CSS</h2>
        </section>
      </form>
    </main>
  </body>
</html>



```

### Expected behavior

Expected a visible change after completing the step, to see an example of what the before pseudo-element does.

### Screenshots

Screenshot 2024-08-03 141211

### System

- Device: Laptop
- OS: Windows 11
- Browser: Chrome
- Version: 127.0.6533.89

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by eric-do**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #55736

### CODE REVIEW

**Review by gikf — COMMENTED** [review]
Hey,

please change also Step 59, which adds css for `p`, and seed code in further tests.

Thank you and happy coding.

**Comment by eric-do** [comment]
@gikf it's already been changed. All the steps from 34 to 68 had to be updated with the css change, actually. They were all using `p::before`.

**Comment by gikf** [comment]
Hey @eric-do

yes, but [Step 59](https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-accessibility-by-building-a-quiz/step-59) adds another css rule for `p` which does not exist anymore, as it also was replaced by `h3` element:

```css
p {
  margin-top: 5px;
  padding-left: 15px;
  font-size: 20px;
}
```

**Comment by eric-do** [comment]
Ah ok I understand the fix, thanks for clarifying, @gikf :)

It's fixed and I tested that the code accepts `h3` and fails `p` tag. Please let me know if that's what you're looking for 👍

**Review by gikf — COMMENTED** [review]
@eric-do that's the one!

Only thing that's left is change seed code in the remaining to use `h3` instead of `p`

```
h3 {
  margin-top: 5px;
  padding-left: 15px;
  font-size: 20px;
}
```

**Comment by eric-do** [comment]
Thank you for the careful check @gikf !

Let me know if I've missed anything else 👍

File: curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148dcaaf2e8750e6cb4501a.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ dashedName: step-59
 
 # --description--
 
-Make the paragraph elements appear as a higher priority, with the following CSS properties:
+Make the h3 elements appear as a higher priority, with the following CSS properties:
```
Comment (gikf) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148dcaaf2e8750e6cb4501a.md:10]:
```suggestion
Make the `h3` elements appear as a higher priority, with the following CSS properties:
```

File: curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148e5aeb102e3142de026a2.md
Comment (gikf) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148e5aeb102e3142de026a2.md:1]: Lines `304` and `633` still are using `p`. (GitHub UI doesn't let to select these specific lines to make suggestion.)

Comment (eric-do) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148e5aeb102e3142de026a2.md:1]: Sorry I missed that :)

File: curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148dcaaf2e8750e6cb4501a.md (hunk 1 above)
Comment (eric-do) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148dcaaf2e8750e6cb4501a.md]: fixed in recent commit

**Review by gikf — APPROVED** [review]

File: curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148dcaaf2e8750e6cb4501a.md [hunk 2]
```diff
@@ -17,28 +17,28 @@ font-size: 20px;
...
-You should give the `p` element a `padding-left` of `15px`.
+You should give the `h3` element a `padding-left` of `15px`.
 
 ```js
-assert.equal(new __helpers.CSSHelp(document).getStyle('p')?.paddingLeft, '15px');
+assert.equal(new __helpers.CSSHelp(document).getStyle('h3')?.paddingLeft, '15px');
 ```
 
-You should give the `p` element a `font-size` of `20px`.
+You should give the `h3` element a `font-size` of `20px`.
```
[the remark this answers is not in the document]
Comment (eric-do) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148dcaaf2e8750e6cb4501a.md]: Updated instruction and tests on 59 and CSS on following steps.

**Review by huyenltnguyen — APPROVED** [review]
LGTM 👍

**Pull request merged**